quantum/matrix_common.c

```diff
@@ -9,27 +9,22 @@
 #    include "split_common/transactions.h"
 #    include <string.h>
 
```

Comment:
```suggestion
#endif

```
Split keyboard check should end here.  missing this causes an "unterminated ifdef" error, and is why all boards are failing in CI.

Reply:
Fix applied. Thanks!